site/content/blog/kctrl-app-commands-blog.md

```diff
@@ -0,0 +1,509 @@
+Target cluster 'https://34.135.209.41' (nodes: gke-cluster-sm-default-pool-28d3ddcd-gjqs, 2+)
+
+Available apps in namespace 'default'
+
+Name        Status               Since Deploy  Owner                     Age  
+hello-app   Reconcile succeeded  1m            PackageInstall/hello-app  1m  
+simple-app  Reconcile succeeded  22s           -                         1m  
+
+Succeeded
+```
+`kctrl` makes a distinction between App CRs deployed by themselves and the ones created due to a package installation. We can take a closer look at an app by using the `get` command.
+```bash
+$ kctrl app get -a simple-app
+Target cluster 'https://34.135.209.41' (nodes: gke-cluster-sm-default-pool-28d3ddcd-gjqs, 2+)
+
+Namespace         default  
+Name              hello-app  
+Service Account   hello-app-default-sa  
+Status            Reconcile succeeded  
+Owner References  - packaging.carvel.dev/v1alpha1/PackageInstall/hello-app  
+Conditions        - type: ReconcileSucceeded  
+                    status: "True"  
+                    reason: ""  
+                    message: ""  
+
+Succeeded
+```
+In case the app is erroring out, the command displays relevant errors. For instance, an app which is applying a resource with a misspelt "kind" field looks like:
+```bash
+$ kctrl app get -a faulty-app
+Target cluster 'https://34.135.209.41' (nodes: gke-cluster-sm-default-pool-28d3ddcd-gjqs, 2+)
+
+Namespace             default  
+Name                  faulty-app  
+Service Account       default-ns-sa  
+Status                Reconcile failed: Deploying: Error (see .status.usefulErrorMessage for details)  
+Owner References      -  
+Conditions            - type: ReconcileFailed  
+                        status: "True"  
+                        reason: ""  
+                        message: 'Deploying: Error (see .status.usefulErrorMessage for details)'  
+Failing Stage         deploy  
+Useful Error Message  kapp: Error: Expected to find kind '/v1/ConfigMaps', but did not:  
+                      - Kubernetes API server did not have matching apiVersion + kind  
+                      - No matching CRD was found in given configuration  
+
+Succeeded
+```
+It highlights that the deploy stage failed, while displaying the error message.
+
+The `pause` and `kick` commands help users trigger and pause reconciliations for their apps.
+
+Pausing an app stops reconciliation for that particular app, which means that kapp-controller will not try to periodically reconcile the app to ensure the desired state on the cluster.
+```bash
+$ kctrl app pause -a simple-app
+Target cluster 'https://34.135.209.41' (nodes: gke-cluster-sm-default-pool-28d3ddcd-gjqs, 2+)
+
+Pausing reconciliation for App 'simple-app' in namespace 'default'
+Continue? [yN]: y
+
+Pausing reconciliation for App 'simple-app' in namespace 'default'
+
+Succeeded
+```
+On the other hand, the `kick` command triggers a reconciliation for the App CR without waiting for the sync period to elapse.
```

Comment:
nit:  sync period to `lapse`

instead of

 sync period to `elapse`.

Reply:
Adopted!

Reply:
Just googled "elapsed vs lapse", turns out we can use both. Depends if we are counting up or counting down apparently 🤣 
I guess we can keep "elapsed" as it feels more positive? I am fine with either